tests/crates/enarx_exec_tests/src/bin/thread-many.rs

```diff
@@ -2,23 +2,37 @@
 
 use enarx_exec_tests::musl_fsbase_fix;
 
-use std::thread;
-
 musl_fsbase_fix!();
 
-fn main() {
-    for _ in 0..100 {
-        let thread1 = thread::spawn(|| 0);
-
-        let thread2 = thread::spawn(|| 0);
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::thread;
+use std::time::Duration;
 
-        let ret: i32 = thread1.join().unwrap();
-        assert_eq!(ret, 0);
+static GLOBAL_THREAD_COUNT: AtomicUsize = AtomicUsize::new(0);
 
-        let ret: i32 = thread2.join().unwrap();
-        assert_eq!(ret, 0);
+fn main() {
+    for i in 0..100 {
+        GLOBAL_THREAD_COUNT.fetch_add(1, Ordering::SeqCst);
+
+        thread::spawn(move || {
+            // do some work
+            eprintln!("{}-th thread reporting", i + 1);
+            thread::sleep(Duration::from_secs(1));
```

Comment:
Does this increase the time it takes to run the tests compared to today?

Reply:
old:
```
$ time cargo test --features dbg --test integration exec::thread_many -- --exact --nocapture 
[…]
test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 43 filtered out; finished in 3.81s


real	0m4.075s
user	0m3.739s
sys	0m1.985s
```

new:
```
$ time cargo test --features dbg --test integration exec::thread_many -- --exact --nocapture 
[…]
test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 43 filtered out; finished in 3.79s


real	0m4.034s
user	0m3.510s
sys	0m1.709s
```